### PR TITLE
feat: Add method to create error in Maestro

### LINKED ISSUE #15: Add methods send to module Error
With the release of the Maestro version supporting the Error feature, it will be necessary to create functions that communicate with this module:
- createError
- createAttachment
- createScreenshot

It is necessary to send the following parameters:
- taskId
- type (error type)
- message(error)
- stackTrace
- language (javascript/typescript)
- tags

### PR DESCRIPTION
**Pull request by kayqueGovetri**
#15

### CODE REVIEW

File: src/botcity/maestro/sdk.ts [hunk 1]
```diff
@@ -300,10 +295,70 @@ export class BotMaestroSdk {
...
-        console.log({ error })
         throw new Error(error.response.data.message)
       })
     await fs.promises.writeFile(filepath, response.data)
     return response.data
   }
+
+  @ensureAccessToken
+  @catchError
+  async createError (taskId: string, error: Error, tags: object[] = [], screenshot: string = '', attachments: Attachment[] = []): Promise<any> {
```
Comment (hhslepicka) [inline src/botcity/maestro/sdk.ts:306]: `createError` -> `error`

File: src/botcity/maestro/utils.ts [hunk 2]
```diff
@@ -33,3 +39,48 @@ export const catchError = (
...
+}
+
+export const getDefaultTags = async (tags: any): Promise<any> => {
+  const userInfo = os.userInfo()
+  tags['user.name'] = userInfo.username
+  tags['host.name'] = os.hostname()
+  tags['os.name'] = os.platform()
+  tags['os.version'] = os.release()
+  tags['node.version'] = process.version
+  tags['npm.list'] = await npmls()
```
Comment (hhslepicka) [inline src/botcity/maestro/utils.ts]:
```suggestion
  tags['user_name'] = userInfo.username
  tags['host_name'] = os.hostname()
  tags['os_name'] = os.platform()
  tags['os_version'] = os.release()
  tags['node_version'] = process.version
  tags['npm_list'] = await npmls()
```

**Review by hhslepicka — APPROVED** [review]

**Comment by hhslepicka** [comment]
@kayqueGovetri talking with @ezambomsantana we decided that it is best to use `_` and on the backend, we will replace all `.` with `_`.

**Comment by hhslepicka** [comment]
When talking with @gabrielarchanjo we decided that it is best to send the module list as an attachment.
I did the same for the Python SDK. We need to reproduce that here as well before merging.

**Review by hhslepicka — APPROVED** [review]
The changes look good. My only concern is that the `npmlist.txt` file is never being deleted but that is a minor concern.

Feel free to merge it and in case this becomes an issue we can address it later.

**Comment by kayqueGovetri** [comment]
> The changes look good. My only concern is that the `npmlist.txt` file is never being deleted but that is a minor concern.
> 
> Feel free to merge it and in case this becomes an issue we can address it later.

When npmlist.txt is created there is a finally that deletes it at the end:
``` fs.unlinkSync(npmListPath) ```.

At least always at the end after creating and uploading the npmlist to the platform it will be deleted.

**Review by hhslepicka — APPROVED** [review]

**Pull request merged**